Sources/PitchSpeller/Data Structures/_Graph.swift

```diff
@@ -82,7 +82,14 @@ struct _Graph<Weight: Weightedness, Pair: SymmetricPair & Directedness & Hashabl
     }
     
     func neighbors (of source: Node) -> [Node] {
-        
+        return nodes.compactMap{
```

Comment:
For some reason, my comment on this seemed to disappear. This might be a little cleaner:

```Swift
return nodes.compactMap { destination in
    adjacents.keys.contains(Pair(source,destination)) ? destination : nil
}
```

Reply:
Also, would the result here be better modeled as `Set<Node>` rather than `Array<Node>`?